<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnForm</class>
 <widget class="QWidget" name="ConnForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>309</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
      <item row="5" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Password:</string>
        </property>
        <property name="buddy">
         <cstring>lePasswd</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Will Topic:</string>
        </property>
        <property name="buddy">
         <cstring>leWillTopic</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>ClientId:</string>
        </property>
        <property name="buddy">
         <cstring>leClientId</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Will Msg:</string>
        </property>
        <property name="buddy">
         <cstring>teWillMsg</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="leUser"/>
      </item>
      <item row="7" column="1" colspan="3">
       <widget class="QLineEdit" name="leWillTopic"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leHost">
        <property name="text">
         <string>localhost</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Username:</string>
        </property>
        <property name="buddy">
         <cstring>leUser</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Server(*):</string>
        </property>
        <property name="buddy">
         <cstring>leHost</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="sbPort">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>1883</number>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="lePasswd"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Port(*):</string>
        </property>
        <property name="buddy">
         <cstring>sbPort</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="3">
       <widget class="QTextEdit" name="teWillMsg"/>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="leClientId">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Keepalive:</string>
        </property>
        <property name="buddy">
         <cstring>sbKeepalive</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbKeepalive">
        <property name="suffix">
         <string notr="true">(s)</string>
        </property>
        <property name="maximum">
         <number>3600</number>
        </property>
        <property name="value">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>CleanSess:</string>
        </property>
        <property name="buddy">
         <cstring>cbCleanSess</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="cbCleanSess">
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="connButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnButton">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leHost</tabstop>
  <tabstop>sbPort</tabstop>
  <tabstop>sbKeepalive</tabstop>
  <tabstop>cbCleanSess</tabstop>
  <tabstop>leClientId</tabstop>
  <tabstop>leUser</tabstop>
  <tabstop>lePasswd</tabstop>
  <tabstop>leWillTopic</tabstop>
  <tabstop>teWillMsg</tabstop>
  <tabstop>connButton</tabstop>
  <tabstop>disconnButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>connButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnForm</receiver>
   <slot>onConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>disconnButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnForm</receiver>
   <slot>onDisconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnect()</slot>
  <slot>onDisconnect()</slot>
 </slots>
</ui>
